### ISSUE TITLE
CLI resolver has no documentation

### ISSUE THREAD

**Issue by alerque** [projectfluent/fluent-rs#140]
As I [posted on discourse](https://discourse.mozilla.org/t/what-is-the-most-functional-cli-option-for-using-fluent/45011?u=alerque) I'm looking for a basic CLI to query a message from `*.ftl` files. I finally figured out how to at least compile the `parser` and `resolver` binaries. The former was somewhat self-explanitory as it responded to a `--help` flag, and seems to do what it says on the tin. Pass it a single argument of at FTL file name and get an parsed AST representation back on STDOUT. Okay, but what do I do with resolver? It doesn't show any help messages, clearly is waiting for something on STDIN, but crashes no matter what I pass it.

* [ ] The link in the Readme should be fixed as it currently links to something on crates.io that doesn't exist.
* [ ] A help message of some kind should be added that explains the usage, what input it expects and what output can be achieved.
* [ ] CLI tool usage should be mentioned somewhere, maybe in the [main rust docs](https://docs.rs/fluent)? I don't know where actually, but I don't see it anywhere the project is talked about.

**Comment by zbraniecki** [comment]
yeah, the cli tooling is in a very 0.0.1 stage :) I just didn't have time to put work into it yet.
